docs/architecture/adr-template.md

```diff
@@ -9,20 +9,35 @@
 > A decision may be "proposed" if the project stakeholders haven't agreed with it yet, or "accepted" once it is agreed. If a later ADR changes or reverses a decision, it may be marked as "deprecated" or "superseded" with a reference to its replacement.
 > {Deprecated|Proposed|Accepted} {Implemented|Not Implemented}
 
+
+## Abstract
```

Comment:
It sounds like context may be duplicate here?

Reply:
Context should give a background. Abstract should present the idea very shortly.

Reply:
Moreover Abstract doesn't explain the problem. Context is all about the problem analysis.

Reply:
Moreover Abstract doesn't explain the problem. Context is all about the problem analysis.